community/kernel/src/main/java/org/neo4j/kernel/internal/StoreLocker.java

```diff
@@ -103,4 +107,9 @@ public void release() throws IOException
             storeLockFileChannel = null;
         }
     }
+
+    @Override
+    public void close() throws IOException {
+        release();
```

Comment:
I was just trying to keep the diff as minimal as possible. But since I now have conflicts anyway I an do that.